docs/devel/annotations.md

```diff
@@ -0,0 +1,72 @@
+# KubeVirt specific annotations and labels
+
+As of the time of writing, KubeVirt doesn't expose any user visible annotations
+or labels specific to its resources. That being said, KubeVirt components rely
+on a number of internal annotations and labels set on different resources to
+support its features. They will be listed below.
+
+**Note:** Annotations and labels that are not specific to KubeVirt are documented
+[here](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/).
+
+## kubevirt.io
```

Comment:
This one is not used for ownership. We kind of have the rule to mark all kubevirt components (not vms), with the `kubevirt.io` key. The value (in this case `virt-launcher`), is a more or less optional description. The label can for instance be used to clean the cluster from kubevirt.